foundry.md

```diff
@@ -76,6 +82,12 @@ Foundry Cheat Codes
 module FOUNDRY-CHEAT-CODES
     imports EVM
     imports FOUNDRY-ACCOUNTS
+
+    configuration
+    <cheatcodes>
+        <prevId> .Account </prevId>
+        <prevOrigin> .Account </prevOrigin>
+    </cheatcodes>
```

Comment:
These cells will add to those in #1349 and could be conceptually confusing. Maybe it's better to create a cell that represents the [`Prank`](https://github.com/foundry-rs/foundry/blob/master/evm/src/executor/inspector/cheatcodes/env.rs#L36) struct. Should come in handy for adding the `prank` cheatcode in the future.
Also, notice that the `depth` is missing, which would cause a `startPrank` to be active for external calls of more depth.